### ISSUE TITLE
Please publish onto Maven Central repository

### ISSUE THREAD

**Issue by ikeike443** [mtkopone/scct#43]
Hi,

Our company are using your cool library everyday, we appreciate to you, thank you.

By the way, would you have any plan to publish it onto Maven Central Repository?

Your library is very useful, so everyone may be more happy, If you could publish it onto central.

Regards,
Takafumi Ikeda

**Comment by fommil** [comment]
even a non-snapshot in sonatype would be very useful - including this plugin is a bottleneck to running pretty much any SBT command in my codebases :-(

**Comment by theon** [comment]
+1 for a non SNAPSHOT release

**Comment by rintcius** [comment]
Indeed, a non snapshot release would be really great to speed up sbt interaction!

**Comment by maiflai** [comment]
Resurrecting this thread as it appears to be an open issue.

I'm happy to upload a version to Maven Central using https://docs.sonatype.org/display/Repository/Uploading+3rd-party+Artifacts+to+The+Central+Repository.

Is there any plan yet for a 0.2 release version please?
